### DISCUSSION TITLE
Problem running e2e tests on gitlab runners. Failed to adjust OOM score of renderer with pid

### DISCUSSION THREAD

**Discussion by ovdartem92** [cypress-io/cypress#26175] [CI setup]
Hello everyone,
Cypress 12.7.0 when I try to run tests in gitlab, I get the following error
Is the main problem for now, where the job runner is lacking some permissions/memory setup which was just working fine in concourse
Note that I have tried many different cypress docker images, but I always ended up getting this error
Also I tried to upgrade KUBERNETES memory, and I use command: 
KUBERNETES_MEMORY_REQUEST: 8Gi

```

[2307:0321/160931.993640:ERROR:zygote_host_impl_linux.cc(263)] Failed to adjust OOM score of renderer with pid 2477: Permission denied (13)
libva error: va_getDriverName() failed with unknown libva error,driver_name=(null)
[2477:0321/160932.016062:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.
```

**Comment by al-mart** [comment]
Any updates ?

**Comment by Lory1990** [comment]
I have the same problem!

**Comment by gogoid88** [comment]
Why are there no comments on this thread ?
I am facing this same issue while execution tests on docker with cypress 12.7
zygote_host_impl_linux.cc : failed to adjust oom score of renderer with pid 595 permission denied
failed to create pki/nssdb directory
the docker execution was working fine with earlier cypress versions.

**Reply by Lory1990** [reply]
I am using a dedicated machnine and all works

**Comment by MoatazFarid** [comment]
did anyone found the solution to this? I am also facing it over bitbucket

**Comment by NilPascual** [comment]
Same issue here for cypress 13.2.0

**Comment by babipal** [comment]
Same issue, running Cypress: 12.17.2 on Gitlab with image: cypress/browsers:node18.12.0-chrome106-ff106

**Reply by babipal** [reply]
I'll add that the same exact test code/Docker setup was working fine on Jenkins CI. Moving over to Gitlab and started seeing this issue.

**Comment by racosgrove** [comment]
Same problem with Gitlab, SaS runners and private K8 runners.  Cypress  12 - 13.6.3.  Tests are chronically unreliable

**Comment by sharmilajesupaul** [comment]
I'm seeing this in k8s on cypress v13.6.6 chrome v120.0.6099.129 ubuntu 22.04
```
[315:0305/164157.674164:ERROR:zygote_host_impl_linux.cc(273)] Failed to adjust OOM score of renderer with pid 474: Permission denied (13)",
```

**Comment by mandarkslab** [comment]
I'm experiencing the same issue on K8s on Cypress v13.6.3 using electron chromium 114

**Comment by NikkiMather** [comment]
Same issue here... Literally cannot run our e2e tests anymore, happens specifically when running on Bitbucket.

**Comment by smallgiantbloo** [comment]
same here, and it only fails on Kubernetes-runners. Docker-runners works fine.

**Reply by smallgiantbloo** [reply]
finally figured it out. In my case I had either .npmrc file or environment variables that defined proxy.
I removed these proxy-configurations from .npmrc (localhost in no_proxy/NO_PROXY did **not** do the trick).
```
proxy=xxxxx
https-proxy=xxxxx
noproxy=xxxxx
```
or
```
export HTTPS_PROXY
export HTTP_PROXY
export NO_PROXY
```

and cypress started working with the kubernetes-runners. In docker-runner running with the proxy settings active worked fine. Turned out the permission denied error was just a red herring in my case.

**Reply by Aaron-Initialyze** [reply]
where to remove the above variables from? can you provide bit more info?

**Reply by smallgiantbloo** [reply]
In my case I could unset the environment variables in the gitlabci job.
```
script:
    - export HTTPS_PROXY=
    - export HTTP_PROXY=
    - export NO_PROXY=
    - <run cypress>
```
I just had to make sure that there's no proxies set on the cypress run because the defined no_proxy/noproxy didn't seem to work.

**Reply by monkpit** [reply]
This seems unrelated to the error message and doesn't seem to fix this issue.

For context, I am also running on self-hosted gitlab with K8s runners.

**Comment by nemonemi** [comment]
How is this a discussion and not a bug or a memory issue topic?

We are experiencing the same thing.

**Comment by rkclark** [comment]
Also seeing this in our Gitlab runners.

**Comment by drdreo** [comment]
We encountered this also when switching from Electron to Chrome in Github Actions

**Comment by godlanbo** [comment]
any updates? I have the same problem!

**Comment by MatthewRyanRead** [comment]
Related issue is #29563

**Comment by jennifer-shehane** [comment]
Please comment in the issue: https://github.com/cypress-io/cypress/issues/29563 where potential bugs are tracked instead of a Discussion in the future.
